src/main/java/io/zksync/sdk/zkscrypto/lib/ZksCrypto.java

```diff
@@ -115,4 +115,10 @@ public ZksSignature signMessage(final ZksPrivateKey privateKey, byte[] message)
         }
     }
 
+    public boolean verifyMessage(final ZksPackedPublicKey publicKey, final ZksSignature signature, final byte[] message) throws Exception {
```

Comment:
Let's stick to the name `verifyMusig` to distinguish it from other signature formats and verifications

Reply:
Think `signMessage` should be renamed too, right?

Reply:
Sure, let's name it `signMusig` for consistency